config.json

```diff
@@ -305,7 +306,12 @@
       "slug": "palindrome-products",
       "difficulty": 1,
       "topics": []
-    }
+    },
+	{
+	  "slug": "twelve-days",
+	  "difficulty": 1,
+	  "topics": []
+	}
```

Comment:
Oops, indents in this file seem a little wonky; please see if you can tweak them to be consistent with the existing content!